brave/brave-ios
# Fix #7120: Remove legacy logins.db and local authentication storage

## Summary of Changes

The PR aims to remove the local database used for login data. After converting the Login/Credential data be stored and used with Chromium APIs, the purpose of this cumbersome implementation of manually encrypted lost its value.
The database is only used for storing Basic Authentication Credentials which is not needed to be stored for arbitrary application launches.

In addition to clean-up the purpose of removal is to have a relief for the problem this database might be causing when application is trying to retrieve data and actually stuck while doing it, especially when opening new external urls and waking up with browser lock feature.



This pull request fixes #7120

Security Ticket: https://github.com/brave/security/issues/1238

## Submitter Checklist:


## Test Plan:


- Open a website that needs Basic Authentication
- Enter credentials and check everything is working with false and true credentials
-Create a new tab and open the same website and check Login Credentials is not needed

## Screenshots:


https://user-images.githubusercontent.com/6643505/228941302-3f4b6f1e-f2a1-4715-bb18-45ba3a09be28.MP4


## Reviewer Checklist:

  - `QA/(Yes|No)`
  - `bug` / `enhancement`
